internal/provider/provider.go

```diff
@@ -23,8 +24,9 @@ type AponoProvider struct {
 	// version is set to the provider version on release, "dev" when the
 	// provider is built and ran locally, and "test" when running acceptance
 	// testing.
-	version string
-	client  *apono.APIClient
+	version        string
+	client         *apono.APIClient
+	internalClient *aponoapi.APIClient
```

Comment:
maybe call it terraformClient? or tfApi?
not sure internal is very clear

Reply:
will change to `terraformClient`